script/Ciel/Day 13/13_01_01_CIEL13_5.txt

```diff
@@ -228,7 +228,7 @@ Her body will not die unless Roa is annihilated.
 Paired with that body is a burning will to fight,
  the flames of which have only been fanned higher by the appearance of the vampire before us――Noel.
  Even though she was once Senpai's partner in exterminating vampires,
- this woman has transformed into one herself, and has already spilled the blood of untold victims.
+ she has now transformed into one herself, having already spilled the blood of untold victims.
```

Comment:
Prefer `and has` over `having`, latter makes it sound like it was a prerequisite for turning into a vampire when I think it's more just that she's done it.

Reply:
I think this might just be like a style difference between us, honestly.
That said, I'll change it back, it's not much of a difference.